mono/utils/win64.asm

```diff
@@ -30,7 +30,22 @@ mono_context_get_current PROC
 	mov [rcx + 68h], r13
 	mov [rcx + 70h], r14
 	mov [rcx + 78h], r15
-
+    movaps [rcx + 090h], xmm0
```

Comment:
Same thing on the comment on mono-context.h, if movups requires less assumptions, we should use it.